stock_change_qty_reason/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Change Quantity Reason
+============================
+
+This module was written to extend the product stock management. It allows to set a reason
```

Comment:
This module extends the product stock management and allows to set a reason
in the wizard when changing the product quantity.